src/win/udp.c

```diff
@@ -1083,7 +1083,7 @@ int uv__udp_connect(uv_udp_t* handle,
 
 int uv__udp_disconnect(uv_udp_t* handle) {
     int err;
-    struct sockaddr addr;
+    struct sockaddr_storage addr;
```

Comment:
What was the issue on windows?

Reply:
Will failed with UV_EFAULT

Reply:
That should be caused by winsock kernel check sockaddr len less than `sizeof(sockaddr_in6)`

Reply:
`The sockaddr structure pointed to by the name contains incorrect address format for the associated address family or the namelen parameter is too small. This error is also returned if the sockaddr structure pointed to by the name parameter with a length specified in the namelen parameter is not in a valid part of the user address space.`
From msft docs: https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-connect